tests/plugins/pastetools/filter/rtf.js

```diff
@@ -0,0 +1,119 @@
+/* bender-tags: editor,pastetools */
+/* bender-ckeditor-plugins: wysiwygarea,pastetools,font,toolbar */
+/* bender-include: ../_helpers/ptTools.js */
+/* global ptTools */
+
+( function() {
+	'use strict';
+
+	bender.editor = {
+		config: {
+			removePlugins: 'pastefromword,pastefromlibreoffice,pastefromgdocs',
+			allowedContent: true
+		}
+	};
+
+	var tests = {
```

Comment:
Test checking for non-existing group extraction could be added to cover this case too.